pallets/nft/src/tests.rs

```diff
@@ -3,6 +3,100 @@ use crate::mock::{Event, *};
 use frame_support::{assert_noop, assert_ok};
 use sp_runtime::AccountId32;
 
+#[test]
+fn demostration_of_event_filter() {
```

Comment:
The issue is, you are testing the test helper, so at least move these to another file. But still I think there should be a better way, though I have no better ideas. Can we specify these test to be run first and if failed, not to proceed? Otherwise after some time the debug will be messy when there is some problem originated from these helpers

Reply:
I can write some atomic test that not use new_test_ext() method. Only to demonstrate the comparison of different event type. But this may not be sensible enough.

Reply:
test_helper reloacted.